package.json

```diff
@@ -40,6 +40,11 @@
       "react-app/jest"
     ]
   },
+  "jest": {
+    "moduleNameMapper": {
+      "axios": "axios/dist/node/axios.cjs"
+    }
+  },
```

Comment:
This allows us to run tests on code that imports `axios` and jest knows where to get `axios` from.